ilc/client.js

```diff
@@ -1,3 +1,4 @@
+import './client/navigationEvents';
```

Comment:
It is a little bit tricky
I think that would be easy just to import `setupEvents` here

Reply:
The idea is to have single entry point to this "module"